kaiba/casting/_cast_to_integer.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import annotations
```

Comment:
@thomasborgen This is nice. I didn't see this before. I learnt something new today :)

Reply:
@ChameleonTartu Great!, its super convenient as it lets you use python 3.10+ annotations like `str | int` or `str | None` instead of having to use `Union[str, int]` or `Optional[str]` !